epoxy-processor/src/main/java/com/airbnb/epoxy/DataBindingModelInfo.kt

```diff
@@ -10,13 +10,13 @@ internal class DataBindingModelInfo(
         private val typeUtils: Types,
         private val elementUtils: Elements,
         val layoutResource: ResourceValue,
-         val moduleName: String,
+        val moduleName: String,
         private val layoutPrefix: String = ""
 ) : GeneratedModelInfo() {
     private val dataBindingClassName: ClassName
 
-    val dataBindingClassElement: Element?
-        get() = getElementByName(dataBindingClassName, elementUtils, typeUtils)
+    private val dataBindingClassElement: TypeElement?
+        get() = getElementByName(dataBindingClassName, elementUtils, typeUtils) as TypeElement?
```

Comment:
should this be `as?` instead?

Reply:
@elihart I'm thinking: if `getElementByName` doesn't return a `TypeElement` or `null` then something unexpected is happening since we're passing it a `ClassName`. In fact maybe it would make more sense to change the return type of `getElementByName(ClassName name, Elements elements, Types types)`. Thoughts?

Reply:
I changed the return type to `TypeElement` and everything still builds and all tests are passing so seems reasonable.